modules/update-duration-example.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/understanding-openshift-update-duration.adoc
+
+:_content-type: CONCEPT
+[id="update-duration-example_{context}"]
+= Example update duration of cluster Operators
+
+image::update-duration.png[A diagram displaying the periods during which cluster Operators update themselves during an OCP update]
+
+The previous diagram shows an example of the time that cluster Operators might take to update to their new versions.
+The example is based on a three-node AWS OVN cluster, which has a healthy compute `MachineConfigPool` and no workloads that take long to drain, updating from 4.13.0 to 4.14.0.
```

Comment:
[The run we timelined][1] was [4.13.0-0.ci-2023-07-31-121322 to 4.14.0-0.ci-2023-07-31-212944][2].  Can we say "from 4.13 to 4.14" here, without pinning specific patch releases?

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.14-upgrade-from-stable-4.13-e2e-aws-ovn-upgrade/1686131647159734272
[2]: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.14-upgrade-from-stable-4.13-e2e-aws-ovn-upgrade/1686131647159734272/artifacts/e2e-aws-ovn-upgrade/gather-extra/artifacts/clusterversion.json

Reply:
Sure thing, I mentioned the z stream in anticipation of you or someone else wanting to be more specific. But I have no issue limiting version references to the minor level.